src/02-common-principles.md

```diff
@@ -413,16 +413,21 @@ label, but must be included in file names (similarly to other key names).
 ## Units
 
 All units SHOULD be specified as per [International System of Units](https://en.wikipedia.org/wiki/International_System_of_Units)
-(abbreviated
-as SI, from the French Système international (d'unités)) and can be SI units or
-[SI derived units](https://en.wikipedia.org/wiki/SI_derived_unit).
-In case there are valid reasons to deviate from SI units or SI
-derived units, the units MUST be specified in the sidecar JSON file. In case
-data is expressed in SI units or SI derived units, the units MAY be specified in
-the sidecar JSON file. In case non-standard prefixes are added to SI or non-SI
-units, these non-standard prefixed units MUST be specified in the JSON file.
+(abbreviated as SI, from the French Système international (d'unités)) and can
+be SI units or [SI derived units](https://en.wikipedia.org/wiki/SI_derived_unit).
+In case there are valid reasons to deviate from SI units or SI derived units,
+the units MUST be specified in the sidecar JSON file.
+In case data is expressed in SI units or SI derived units, the units MAY be
+specified in the sidecar JSON file.
+In case non-standard prefixes are added to SI or non-SI units, these
+non-standard prefixed units MUST be specified in the JSON file.
 See [Appendix V](99-appendices/05-units.md) for a list of standard units and
 prefixes.
+Note also that for the *formatting* of SI units, the [CMIXF-12](https://people.csail.mit.edu/jaffer/MIXF/CMIXF-12)
+convention for encoding units is REQUIRED.
+More information about CMIXF can be found in [Appendix V](99-appendices/05-units.md)
+but to provide some examples, CMIXF formatting of "micro volts" would be `uV`,
+and of "kilobecquerel per milliliter" would be `kBq/mL`.
```

Comment:
Well, if we're not in a rush to approve, can I suggest smoother wording here? "... but to provide..." is weird. Better would be to sketch out what CMIXF is trying to do, then provide an example; also, "kBq/mL" isn't so informative b/c that's a routine abbreviation -- better to focus on the exceptions that might be counter intuitive.

What about this text for this passage, starting after "convention for encoding units is REQUIRED."

> CMIXF provides a consistent system for all units and prefix symbols with only basic characters, avoiding symbols that can cause text encoding problems; for example the CMIXF formatting for "micro volts" is `uV`, "degrees Celsius" is `oC` and "Ohm" is `Ohm`. See [Appendix V](99-appendices/05-units.md) for more information.

Happy to make a PR if people like this.

Reply:
> if we're not in a rush to approve

definitely not in a rush, I am just trying to keep the momentum up so that this PR does not go stale for another year. If we see problems or ways to make it better, we should (and will) take the time to make it as good as we can given all constraints.

> Happy to make a PR if people like this.

PRs are welcome to: https://github.com/sappelhoff/bids-specification/tree/asciiunits

an alternative for PRs is to use the [GitHub suggestion function](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/reviewing-proposed-changes-in-a-pull-request) --> multiline suggestions are now also possible [gh-blog-post on this](https://github.blog/changelog/2020-02-26-multi-line-code-suggestions-beta/)

Reply:
Cool!  [Done](https://github.com/bids-standard/bids-specification/pull/411#discussion_r432484588).

Reply:
Thanks, this is a lot better.